src/test/java/SingUpTest.java

```diff
@@ -73,7 +69,23 @@ public void negativeSignUpTestShortPassword () {
             WebElement registerButton= driver.findElement(By.cssSelector("[value='Register']"));
             registerButton.click();
             Assert.assertTrue(driver.findElement(By.className("error_message")).isDisplayed(), "Password consisting of one digit has been validated");
-            driver.quit();
+        }
+        @Test
+        public void positiveTestAddBookToCart () {
+            driver.get("https://www.sharelane.com/cgi-bin/main.py");
+            WebElement testPortalButton = driver.findElement(By.xpath("/html/body/center/table/tbody/tr[6]/td/a[2]"));
```

Comment:
Please do not use full xpath locators, they'r every unstable. Try to find another way (locator) to identify these elements